mix.exs

```diff
@@ -83,20 +83,20 @@ defmodule Exzeitable.MixProject do
   defp deps do
     [
       # Phoenix
-      {:phoenix, "~> 1.3.0 or ~> 1.4.0"},
+      {:phoenix, "~> 1.5.0"},
```

Comment:
There are a few deprecation warnings to address, I would be happy to address them as well.

```Rendering the child template from layouts is deprecated. Instead of:

    <%= render(@view_module, @view_template, assigns) %>

You should do:

    <%= @inner_content %>

  (phoenix) lib/phoenix/view.ex:283: Phoenix.View.render/3
  (exzeitable) lib/exzeitable_web/templates/layout/app.html.eex:14: TestWeb.LayoutView."app.html"/1
  (phoenix) lib/phoenix/view.ex:472: Phoenix.View.render_to_iodata/3
  (phoenix) lib/phoenix/controller.ex:776: Phoenix.Controller.render_and_send/4
  (exzeitable) lib/exzeitable_web/controllers/post_controller.ex:1: TestWeb.PostController.action/2
  (exzeitable) lib/exzeitable_web/controllers/post_controller.ex:1: TestWeb.PostController.phoenix_controller_pipeline/2
  (phoenix) lib/phoenix/router.ex:352: Phoenix.Router.__call__/2
  (exzeitable) lib/exzeitable_web/endpoint.ex:1: TestWeb.Endpoint.plug_builder_call/2
  (exzeitable) lib/exzeitable_web/endpoint.ex:1: TestWeb.Endpoint.call/2
  (phoenix) lib/phoenix/test/conn_test.ex:225: Phoenix.ConnTest.dispatch/5
  test/exzeitable_web/controllers/page_controller_test.exs:9: TestWeb.PageControllerTest."test Pages Get posts page"/1
  (ex_unit) lib/ex_unit/runner.ex:355: ExUnit.Runner.exec_test/1
  (stdlib) timer.erl:166: :timer.tc/1
  (ex_unit) lib/ex_unit/runner.ex:306: anonymous fn/4 in ExUnit.Runner.spawn_test_monitor/4
```